apps/fluent-tester/src/TestComponents/Badge/PresenceBadgeTest.android.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import React, { useState } from 'react';
+import { View, Text } from 'react-native';
+
+import { ToggleButton } from '@fluentui/react-native';
+import { PresenceBadge } from '@fluentui-react-native/badge';
+
+export const PresenceBadgeTest: React.FunctionComponent = () => {
+  const [outOfOffice, setOutOfOffice] = useState(false);
+  return (
+    <View>
+      <ToggleButton onClick={() => setOutOfOffice(!outOfOffice)} checked={outOfOffice}>
+        Set {outOfOffice ? ' In office' : ' Out of office'}
+      </ToggleButton>
+      <Text>Small</Text>
+      <View style={{ flexDirection: 'row' }}>
```

Comment:
don't applying styling directing here, have a object outside render or think there are already util styling for this.  

same for other places.